base/src/main/java/uk/ac/ebi/atlas/trader/ExperimentTrader.java

```diff
@@ -16,7 +16,9 @@ public ExperimentTrader(ExperimentDAO experimentDAO) {
         this.experimentDAO = experimentDAO;
     }
 
-    public abstract Experiment getPublicExperiment(String experimentAccession);
+    public Experiment getPublicExperiment(String experimentAccession) {
+        return getExperiment(experimentAccession, "");
+    }
 
```

Comment:
I don’t understand this change...

Reply:
This is a convenience method. We can implement it right up in ExperimentTrader - despite not knowing what getExperiment will do exactly, we expect it to give us a public experiment with "" for an accessKey, so we do that instead of marking it as abstract and getting the children to all do it in their own slightly different way.